### ISSUE TITLE
Typo3 v9 compatibility

### ISSUE THREAD

**Issue by nollm** [PHORAX/loginusertrack#3]
It would be great to have compatibility with Typo3 v9. Are you planing to do this?

**Comment by 7elix** [comment]
Yes, but currently we focus on other extensions that we maintain/forked as well. So there is no schedule for loginusertrack.

Would you like to support the effort with coding and/or testing?

If yes: please test this version with typo3 9 lts already and send single bug reports / issues or pull requests for each error.

Thank you!
